modules/apps/transfer/module.go

```diff
@@ -208,9 +209,19 @@ func ValidateTransferChannelParams(
 	}
 
 	// Require portID is the portID transfer module is bound to
-	boundPort := keeper.GetPort(ctx)
-	if boundPort != portID {
-		return sdkerrors.Wrapf(porttypes.ErrInvalidPort, "invalid port: %s, expected %s", portID, boundPort)
+	boundPorts := keeper.GetPorts(ctx)
```

Comment:
The benefit of using `GetPorts` as opposed to storing each port under `ports/` key is less migration code?